### PR TITLE
Introduce `CommercialLazyLoadMarginReloaded` test

### PR DESCRIPTION
**Pull request by simonbyford**
## What does this change?

Following on from the success of the previous experiment `CommercialLazyLoadMargin` ([PR](https://github.com/guardian/frontend/pull/24839)) we are introducing a new experiment with different margins. These are:

| Variant   | Margin |
|-----------|--------|
| `control` | 20% of the viewport height    |
| `variant-1` | 0% of the viewport height     |
| `variant-2` | 10% of the viewport height    |
| `variant-3` | 30% of the viewport height    |
| `variant-4` | 40% of the viewport height    |
| `variant-5` | 50% of the viewport height    |
| `variant-6` | 60% of the viewport height    |
| `variant-7` | 70% of the viewport height    |

## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [x] Yes [(please indicate your plans for DCR Implementation)](https://github.com/guardian/dotcom-rendering/pull/5185)

## Screenshots

173536495-3bb0058e-501d-4ed3-b9be-7493c89cea19

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Review by arelra — APPROVED** [review]

File: static/src/javascripts/projects/common/modules/experiments/tests/commercial-lazy-load-margin-reloaded.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import type { ABTest } from '@guardian/ab-core';
+import { noop } from '../../../../../lib/noop';
+
+export const commercialLazyLoadMarginReloaded: ABTest = {
+	id: 'CommercialLazyLoadMarginReloaded',
+	start: '2022-06-20',
+	expiry: '2022-07-04',
+	author: 'Simon Byford',
+	description:
+		'Once again test various margins at which ads are lazily-loaded in order to find the optimal one, this time using values between 0px and 70px',
```
Comment (zekehuntergreen) [inline static/src/javascripts/projects/common/modules/experiments/tests/commercial-lazy-load-margin-reloaded.ts:10]:
```suggestion
		'Once again test various margins at which ads are lazily-loaded in order to find the optimal one, this time using values between 0% and 70% of the viewport height',
```

**Review by zekehuntergreen — APPROVED** [review]
Great test name 😎

File: static/src/javascripts/projects/common/modules/experiments/tests/commercial-lazy-load-margin-reloaded.ts (hunk 1 above)
Comment (simonbyford) [reply static/src/javascripts/projects/common/modules/experiments/tests/commercial-lazy-load-margin-reloaded.ts:10]: Ooh great catch!

**Review by chrislomaxjones — COMMENTED** [review]
Looks good! Only thing I noticed is you might want to opt into sending metrics from Frontend via [shouldCaptureMetrics.ts](https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/analytics/shouldCaptureMetrics.ts#L6)

**Comment by simonbyford** [comment]
> Looks good! Only thing I noticed is you might want to opt into sending metrics from Frontend via [shouldCaptureMetrics.ts](https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/analytics/shouldCaptureMetrics.ts#L6)

Ah good catch! I've made the change

**Review by chrislomaxjones — APPROVED** [review]

**Pull request merged**